source/_patterns/10-atoms/content/media-caption.scss

```diff
@@ -118,7 +119,8 @@
   margin-top: 4px;
   text-transform: none;
 
-  strong {
+  // strong,
+  .media-caption__source-strong {
```

Comment:
Also this rule is identical to the one at Line 105 => Video & Audio haben da gerne ihr eigenes